### PR TITLE
Add clarifying documentation for command-line builds

### PR DESCRIPTION
**Pull request by wilhuff**

### CODE REVIEW

File: Firestore/README.md [hunk 1]
```diff
@@ -10,6 +12,42 @@
...
+ever since Xcode 10.2, CocoaPods generates Xcode projects that are break by
+default when built by the `xcodebuild` command-line tool. There's a workaround
+possible that involves disabling Xcode's default mechanism of finding implicit
+dependencies, but this is something we'd have to disable Firebase-wide and
+there hasn't been an appetite to do this.
+
+### Swift package manager
+
+Firestore also supports building with Swift Package Manager. To build this way
+use:
```
Comment (paulb777) [inline Firestore/README.md:43]: Or `open Package.swift` and choose the FirebaseFirestore scheme to build in Xcode.

Comment (wilhuff) [reply Firestore/README.md:43]: Having never done that, I hesitate to document it. Feel free to push a commit on top that adds that.

Comment (paulb777) [reply Firestore/README.md:43]: OK, I'll either do that or wait until we're ready to address the Firestore SPM technical debt more broadly.

**Review by wu-hui — APPROVED** [review]
Thank you!

**Pull request merged**